src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Model/Column.cs

```diff
@@ -434,6 +434,30 @@ public override void Set(SqlDataRecord record, int ordinal, byte? value)
         }
     }
 
+    public class TimestampColumn : Column<byte[]>
+    {
+        public TimestampColumn(string name)
+            : base(name, SqlDbType.Timestamp, true)
+        {
+        }
+
+        public override byte[] Read(SqlDataReader reader, int ordinal)
+        {
+            // The row version storage size is 8 bytes.
+            const int length = 8;
+
+            byte[] bytes = new byte[length];
+            reader.GetBytes(Metadata.Name, ordinal, 0, bytes, 0, length);
+
+            return bytes;
+        }
+
+        public override void Set(SqlDataRecord record, int ordinal, byte[] value)
+        {
+            record.SetBytes(ordinal, 0, value, 0, value.Length);
+        }
+    }
```

Comment:
[This](https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/sql-server-data-type-mappings) seems to indicate that both timestamp and rowversion data types are mapped to a byte array.

At the same time, [documentation for rowversion](https://docs.microsoft.com/en-us/sql/t-sql/data-types/rowversion-transact-sql?view=sql-server-ver15) specifies:

`A nonnullable rowversion column is semantically equivalent to a binary(8) column. A nullable rowversion column is semantically equivalent to a varbinary(8) column.`

Should behavior here differ based on if the column is specified as nullable or not?

Reply:
@johnstairs I believe you would the best person to answer this. Going to merge this PR to my feature branch for now, but this can be revisited after the holidays if needed.